static/js/tools/shared/metadata/metadata_fetcher.ts

```diff
@@ -35,6 +35,14 @@ import {
  * This module contains helper functions for fetching and processing metadata.
  */
 
+/* 
+  The following are lists of provenances that contain known incorrect or otherwise
+  concerning date ranges or measurement methods. The respective date range or
+  measurement method will not be included in the compiled metadata.
+ */
+const dateRangeSuppressionProvenances = ["Wikidata", "WikipediaStatsData"];
+const measurementMethodSuppressionProvenances = ["WikipediaStatsData"];
```

Comment:
super nit: since these are constants, please update to use all caps for the constant names:

https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
please also add a todo that these are temporary (and should be fixed upstream in the data source)